src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/_k8s_util.py

```diff
@@ -299,6 +299,16 @@ def delete_replica_set(self, deployment_name):
             print("Unable to delete replica set for deployment {}. {} {}".format(deployment_name, exc, exc.output))
             raise
 
+    def scale_deployment(self, service_name, num_replicas):
+        try:
+            print("Scaling web service {} to {} pods".format(service_name, num_replicas))
+            deployment_name = service_name + '-deployment'
+            num_replicas = int(num_replicas)
+            subprocess.check_call(['kubectl', 'scale', 'deployment', deployment_name,
+                                   '--replicas={}'.format(num_replicas)])
```

Comment:
as discussed in chat, add a print statement to indicate that increasing pods may cause list/view to show as unhealthy.